divi/src/test/sanity_tests.cpp

```diff
@@ -7,13 +7,19 @@
 
 #include <boost/test/unit_test.hpp>
 #include "test_only.h"
+
 BOOST_AUTO_TEST_SUITE(sanity_tests)
 
-BOOST_AUTO_TEST_CASE(basic_sanity,SKIP_TEST)
+BOOST_AUTO_TEST_CASE(basic_sanity)
 {
+  ECCVerifyHandle verificationModule;
+  ECC_Start();
+
   BOOST_CHECK_MESSAGE(glibc_sanity_test() == true, "libc sanity test");
   BOOST_CHECK_MESSAGE(glibcxx_sanity_test() == true, "stdlib sanity test");
   BOOST_CHECK_MESSAGE(ECC_InitSanityCheck() == true, "openssl ECC test");
```

Comment:
This is the line that needs to be bracketed by the instantiation of the verification module as well as the ECC_Start()/ECC_Stop(). It makes it clearer what parts of the test depend on that.

Reply:
Line 20 that is

Reply:
In that case, put `ECC_Start()` and` ECC_Stop()` inside the `ECC_InitSanityCheck()` function?

Reply:
No, that won't work. ECC_Start() instantiates the global & ECC_Stop() clears it, unfortunately.